patching/src/main/java/org/jboss/as/patching/management/PatchStreamResourceOperationStepHandler.java

```diff
@@ -40,6 +40,7 @@
  * @author Alexey Loubyansky
  */
 abstract class PatchStreamResourceOperationStepHandler implements OperationStepHandler {
+    protected boolean acquireWriteLock = true;
```

Comment:
Nit: It looks like this can be package protected.

I hate 'protected' due to too many cases where I couldn't improve some piece of code because some external code might have been accessing a protected member. That's not possible here but it's kind of a code standard thing for me not to use it if it's not required.

Reply:
Nice tip, I've just removed the `protected` modifier.